src/packageManager/FileDownloader.ts

```diff
@@ -90,15 +89,12 @@ async function downloadFile(fd: number, description: string, urlString: string,
             });
 
             response.on('end', () => {
-                resolve();
+                resolve(Buffer.concat(buffers));
             });
 
             response.on('error', err => {
                 reject(new NestedError(`Reponse error: ${err.message || 'NONE'}`, err));
```

Comment:
I hadn't noticed this before. This exception isn't super descriptive. Should it be somethigng like `Failed to download from ${uri}. Error Message: $err.message || 'NONE'}`?